README.md

```diff
@@ -3,9 +3,25 @@ Medic Android App
 
 The medic-android application is a thin wrapper to load the [CHT Core Framework](https://github.com/medic/cht-core/) web application in a webview. This allows the application to be hardcoded to a specific CHT deployment and have a partner specific logo and display name. This app also provides some deeper integration with other android apps and native phone functions that are otherwise unavailable to webapps.
 
+
+# APKs
+
+For compatibility with a wide range of devices the build script produces multiple APKs. The two variables are the architecture of the device's CPU, and the supported Android version. When publishing to the Google Play Store upload all APKs and it will automatically choose the right one for the target device. However, when sideloading the application it is essential to pick the correct APK or the application may crash.
```

Comment:
Maybe replace _"architecture"_ with _"instruction set"_ or _"instruction set (architecture)"_ to align the text with the content above.

Reply:
Done. Thanks!